src/app/clients/view-client/view-client.component.ts

```diff
@@ -75,9 +78,11 @@ export class ViewClientComponent implements OnInit, OnDestroy {
         (res => {
           const groupArray = new Array;
           this.clientInfo = res;
-          this.clientInfo.dateOfBirth = this.clientInfo.dateOfBirth.reverse().join('-');
+          if (this.clientInfo.imagePresent) {
+            this.getClientImage(this.id);
+          }
+          // this.clientInfo.dateOfBirth = this.clientInfo.dateOfBirth.reverse().join('-');
```

Comment:
Unrelated change.

Reply:
I should suggest that you should keep this change as for some of the clients whose DOB is not present in the db, it throws the error as mentioned in #363 , so when the exception is raised on the component, the rest of the code below this line isn't executed which made it frustrating while debugging the image code i've fetched.

Reply:
This error isn't supposed to be resolved as part of the issue you are working on.